warehouse/templates/manage/account.html

```diff
@@ -662,7 +669,7 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
       {% endif %}
     {%- endmacro %}
 
-    <p>{% trans %}Events appear here as security-related actions occur on your account.{% endtrans %}</p>
+    <p>{% trans %}Events appear here as security-related actions occur on your account. If you notice anything suspicious, please <a href="/help#suspicious-activity">secure your account</a> as soon as possible.{% endtrans %}</p>
```

Comment:
We should use `request.help_url` to generate the URL here. Also see https://github.com/pypa/warehouse/issues/8040#issuecomment-652076266 for an example of passing this into the translated string.